firmware-binaries/.cargo/config.toml

```diff
@@ -4,7 +4,11 @@
 
 [build]
 target = "riscv32imc-unknown-none-elf"
-rustflags = ["-C", "target-feature=+f"]
+rustflags = [
+  "-C", "target-feature=+f",
+  "-C", "force-frame-pointers=yes",
+  "-C", "force-unwind-tables=yes"
```

Comment:
Could you add an `XXX: Seem to be ignored`?

Reply:
Yep, can do.

Reply:
Wait the other flags _also_ seem to be ignored write? When we add gooblygook there, we don't get any warnings or errors.